<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2021-04-16T01:55:09" hostname="DESKTOP-3R4DVF7" time="0.058">
  <properties/>
  <testcase name="negativeTimeout" classname="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" time="0.0"/>
  <testcase name="timeoutExceptionThrownIfNoUserException" classname="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" time="0.053"/>
  <testcase name="nullNextStatement" classname="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" time="0.0"/>
  <testcase name="userExceptionPropagates" classname="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" time="0.004"/>
  <testcase name="noExceptionThrownIfNoUserExceptionAndTimeoutDoesNotOccur" classname="org.springframework.test.context.junit4.spr16716.SpringFailOnTimeoutTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
